programs/create-genesis/genesis_create.cpp

```diff
@@ -665,11 +665,7 @@ struct genesis_create::genesis_create_impl final {
                 _exp_info.balance_events.push_back(record("account", account));
             } else {
                 auto& acc_info = _exp_info.account_infos[*acc_id];
-                if (pk == gls_pk) {
-                    acc_info["balance"] = balance;
-                } else {
-                    acc_info["balance_in_sys"] = balance;
-                }
+                acc_info[pk == gls_pk ? "balance" : "balance_in_sys"] = balance;
```

Comment:
i dislike ternaries

Reply:
in this case it removes 4 LOC and makes code clearer. also it makes impossible to change one `if` branch and forget to change another. is it enough vs. "dislike"?

Reply:
@zxcat it removes 4 LOC in any case :)
ok, we can use ternary here because `balance` is same in 2 branches

Reply:
>  it removes 4 LOC in any case :)

nop. if ternary if too cryptic, it's better to separate it like:
```cpp
auto balance_type = (pk == gls_pk) ? "balance" : "balance_in_sys";
acc_info[balance_type] = balance;
```
of even
```cpp
auto balance_type = (pk == gls_pk) ?
    "balance" :
    "balance_in_sys";
```

> because `balance` is same in 2 branches

not only `balance`. Also `acc_info`, `[]` and `=` (e.g. change to `+=`)

Reply:
@zxcat 

> e.g. change to +=

i like it